src/Liip/RMT/Output/Output.php

```diff
@@ -117,6 +117,11 @@ public function writeEmptyLine($repeat = 1)
     // QuestionHelper does about the same as we do here.
```

Comment:
@ragusa87 Have you seen that comment? I guess you are doing something similar...